egnoramoose.py

```diff
@@ -1,119 +1,167 @@
 class Pawn:
-	def __init__(self,location):
-		self.location = location
+    pass
+
+
+class Location:
+    def __init__(self, *location):
+        self.row = -1
+        self.column = -1
+        self._convert(location)
+
+    def __sub__(self, other):
```

Comment:
This is how python allow us to subtract our objects and return something. This triggers when I do object1 - object2